### PR TITLE
kubewarden-save-images.sh and kubewarden-load-images.sh

### LINKED ISSUE #2: kubewarden-save-images.sh and kubewarden-load-images.sh
These two scripts are needed for airgap installation. We can't reuse the scripts provided by rancher because registry is not replaced from the name. This is because `rancher-images.txt` does not contain the registry as it is docker.io (e.g. rancher/rke-tools:v0.1.87). We need the registry for our images (e.g `ghcr.io/kubewarden/kubewarden-controller:v1.3.0`). 

If we use the rancher script with localhost:5000 as a private registry:
`ghcr.io/kubewarden/kubewarden-controller:v1.3.0` is pushed as `localhost:5000/ghcr.io/kubewarden/kubewarden-controller:v1.3.0`. This is not correct it should be `localhost:5000/kubewarden/kubewarden-controller:v1.3.0`

`kubewarden-save-images.sh` saves images into a tar.gz
`kubewarden-load-images.sh` load images from a tar.gz into a private repository

### PR DESCRIPTION
**Pull request by raulcabello**
`kubewarden-save-images.sh` saves container images into a tar.gz
`kubewarden-load-images.sh` loads container images from a tar.gz

This is needed for the airgap installation.

Fix #2

### CODE REVIEW

File: .github/workflows/shellcheck.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Airgap
```
Comment (jvanz) [inline .github/workflows/shellcheck.yml]: Is not a name like `shellcheck` or `ci` a better one?

**Review by flavio — APPROVED** [review]
LGTM. The real question is: do we actually need it? Can't we just point people to the bash scripts provided by Rancher Manager?

**Comment by raulcabello** [comment]
@flavio yes, we need different scripts. [More info](https://github.com/kubewarden/utils/issues/2#issue-1453448131)

File: .github/workflows/shellcheck.yml (hunk 1 above)
Comment (raulcabello) [reply .github/workflows/shellcheck.yml]: done in https://github.com/kubewarden/utils/pull/3/commits/e4da1df466ce3ff1a27f4e07f881ea47363c1932

**Review by viccuad — APPROVED** [review]
LGTM

**Pull request merged**